### PR TITLE
Merge to upstream and fix the used warning

### PR DESCRIPTION
**Pull request by nick863**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by jhakulin — APPROVED** [review]

File: sdk/ai/azure-ai-projects/azure/ai/projects/aio/_patch.py [hunk 1]
```diff
@@ -196,18 +201,41 @@ def __init__(  # pylint: disable=super-init-not-called,too-many-statements
...
         self.evaluations = EvaluationsOperations(self._client3, self._config3, self._serialize, self._deserialize)
         self.inference = InferenceOperations(self)
 
+    async def _close_credential_client(self):
+        """
+        Close credential pipeline clients.
+
+        To authenticate to azure we are using pipeline mechanism. If the credential is async,
+        azure.identity opens new aio session, which is dedicated to renew token. When we exit
+        program, this session is not being appropriately closed, causing thw warning about it.
```
Comment (dargilco) [inline sdk/ai/azure-ai-projects/azure/ai/projects/aio/_patch.py:222]: typo (thw)

File: sdk/ai/azure-ai-projects/azure/ai/projects/aio/_patch.py [hunk 2]
```diff
@@ -196,18 +201,41 @@ def __init__(  # pylint: disable=super-init-not-called,too-many-statements
...
+        In the code below we are going through all credentials in AsyncTokenCredential and close all
```
Comment (dargilco) [inline sdk/ai/azure-ai-projects/azure/ai/projects/aio/_patch.py:223]: "and closing"

**Comment by dargilco** [comment]
Please also update CHANGELOG.md with the following, put it at the top and describe your fix:

## 1.0.0b2 (Unreleased)

### Bugs Fixed

* <your text here>


Also please update `\azure\ai\projects\_version.py` to version 1.0.0b2

**Review by dargilco — APPROVED** [review]

File: sdk/ai/azure-ai-projects/azure/ai/projects/aio/_patch.py (hunk 2 above)
Comment (nick863) [reply sdk/ai/azure-ai-projects/azure/ai/projects/aio/_patch.py]: Fixed.

File: sdk/ai/azure-ai-projects/azure/ai/projects/aio/_patch.py (hunk 1 above)
Comment (nick863) [reply sdk/ai/azure-ai-projects/azure/ai/projects/aio/_patch.py]: Fixed.

**Pull request closed without merging**